src/Components/EventContainer.tsx

```diff
@@ -26,9 +22,42 @@ const EventContainer = () => {
     )
   })
 
+  const filteredEventCards = filteredEvents && filteredEvents.map((event: any, index: any) => {
+    return (
+      <Opp
+        key={index} 
+        event={event}
+      />
+    )
+  })
+
+  const displayedEventCards = () => {
+    return !filteredEvents.length && category ? 'Sorry, no events available with that category selection!'
```

Comment:
great logic here Adam. This is an awesome start to our filtered events functionality.